services/prometheus/scrape_configs.go

```diff
@@ -142,7 +143,7 @@ func scrapeConfigForStandardExporter(interval time.Duration, node *models.Node,
 	}
 
 	cfg := &config.ScrapeConfig{
-		JobName:        jobName(agent),
+		JobName:        fmt.Sprintf("%s_%s", jobName(agent), interval),
```

Comment:
I would let `jobName` function return a job name, not a part of it. Add `interval` as a parameter to it.